src/esperanza/walletextension.cpp

```diff
@@ -248,8 +252,7 @@ bool WalletExtension::SetMasterKeyFromSeed(const key::mnemonic::Seed &seed,
 }
 
 bool WalletExtension::BackupWallet() {
-  const std::string wallet_file_name = m_enclosing_wallet.GetName().empty() ?
-          "wallet.dat" : m_enclosing_wallet.GetName();
+  const std::string wallet_file_name = m_enclosing_wallet.GetName().empty() ? "wallet.dat" : m_enclosing_wallet.GetName();
```

Comment:
I am looking at this changed line and for the life of me I do not see the difference

Reply:
Like shaggy would say, it wasn't me... I'll revert whatever happened here, didn't even notice.

Reply:
Yeah so it was some pre commit hook, I disabled it and changed back.